.gitignore

```diff
@@ -1,12 +1,28 @@
-/.idea/
-.DS_Store
-.gradle
-/local.properties
-/build
-/captures
-.externalNativeBuild
-.cxx
-local.properties
```

Comment:
Question: Why do you removed some files in the gitignore ?
List of file removed:
- .gradle
- /local.properties
- /build
- /captures
- .externalNativeBuild
- .cxx
- local.properties

Reply:
I think this might just be due to a difference in placement for the files compared to the main branch, if you look at the gitignore on my branch, the files are still there